enhancements/microshift/microshift-default-csi-plugin.md

```diff
@@ -0,0 +1,345 @@
+
+> _Assuming a [RHEL 8.x](https://github.com/openshift/microshift/blob/main/docs/devenv_rhel8_auto.md) or 
+> [RHEL for Edge](https://github.com/openshift/microshift/blob/main/docs/rhel4edge_iso.md) host, ..._
+
+1. The device owner will connect to the device terminal.
+1. If the MicroShift service is not running, the device owner will start the service via `systemctl enable --now microshift`.
+1. The device owner will verify a stable deployment (see: **Validate-ODF-LVM function**)
+
+**Validate ODF-LVM function**
+1. The device owner will observe the cluster to confirm ODF-LVM pods are running with `oc get pods -n openshift-storage`.
+
+    ```shell
+    $ oc get pods -n openshift-storage
+    NAME                                  READY   STATUS    RESTARTS        AGE
+    topolvm-controller-8479455f95-tdv7f   4/4     Running   0               30m
+    topolvm-node-clv2z                    4/4     Running   0               30m
+    ```
+
+2. The device owner will create a simple Pod and PersistentVolumeClaim to test-drive the plugin with:
+    ```shell
+    cat <<'EOF' | oc apply -f -
+    kind: PersistentVolumeClaim
+    apiVersion: v1
+    metadata:
+      name: my-lv-pvc
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 1G
+    ---
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: my-pod
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+        command: ["/usr/bin/sh", "-c"]
+        args: ["sleep", "1h"]
+        volumeMounts:
+        - mountPath: /mnt
+          name: my-volume
+      volumes:
+        - name: my-volume
+          persistentVolumeClaim:
+            claimName: my-lv-pvc
+    EOF
+    ```
+
+**Maintainer Driven Rebase**
+
+Because MicroShift is based on OCP, it must ensure parity between its cluster components and those of OCP, by version.
+OCP records cluster component image digests in the ocp-release image, which is versioned according to its corresponding
+OCP version.  ODF maintains a similar release image called ocs-release, which maps ODF-LVM image digests to OCP versions.
+The workflow is automated and documented 
+[here](https://github.com/openshift/microshift/blob/9dfddbd6801098c96bbd7db887c2844cad177c69/docs/rebase.md).
+
+#### Variation
+
+**Start MicroShift with a user-provided ODF-LVM configuration**
+
+> After the device owner has logged into the device ... 
+
+1. The device owner will create a file called `/etc/microshift/lvmd.yaml`
+1. The device owner will define the lvmd config, according to its 
+[documentation](https://github.com/red-hat-storage/topolvm/blob/main/docs/lvmd.md)
+
+   _For example:_
+   
+    ```shell
+    $ cat <<EOF > /etc/microshift/lvmd.yaml
+    socket-name: 
+    deviceClasses:
+    - default: true
+      name: "user-provided-config"
+      volume-group: rhel
+      spare-gb: 10 # in GB
+    socket-name: /run/lvmd/lvmd.sock # socket-name must always be /run/lvmd/lvmd.sock
```

Comment:
@nbalacha - why do we have this option, if it always needs to be set to this precise value? Can we remove it or at least default to that value?

Reply:
My assumption here is that it supports lvmd when run as a system daemon on a host.  Running in a pod, it's less useful.